### ISSUE TITLE
:wheelchair: a11y(productslider): sequence of elements

### ISSUE THREAD

**Issue by carolabes** [baloise/design-system#1418]
### Description of this issue

* Cards that are not visible can be selected using the keyboard and screen reader. 
* The navigation arrow is only navigated to at the end. 

<img width="1504" alt="productslider-linklist" src="https://github.com/baloise/design-system/assets/172396366/1faebb12-c7a3-44ef-9f0b-98ef8c2e8e20">

### How to fix it

* Only the visible elements should be selected and then the navigation arrow, so that the arrow can be used to navigate further

### Additional Information

Level A
https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hirsch88** [comment]
ready to be released with the next version after 16.3.0

**Comment by carolabes** [comment]
Issue still exists on Baloise.ch 
navigation arrows are aria-hidden=true as well and not accessible with screenreader

**Comment by GinaBiondo** [comment]
Hey @hirsch88, why is this issue closed? The controls are still not accessible via keyboard. There is a tabIndex="-1" on both controls and aria-hidden does not update with the button visibility. Slides that are hidden are accessible via tab, which updates the slider position as well but I am not sure if this is correct accessibility-wise.

**Comment by GinaBiondo** [comment]
Hey @hirsch88, we've checked for some possible solutions for the current tabbing behavior (tabbing through all items).

### First recommended approach: 

slider controls are the first element to focus. afterwards teasers (typical carousel behavior )
only visible teasers are focusable. if the user wants to see more => use slider controls
therefore the teasers are easily skippable 
Examples: Carousel 'Neues im Magazin": https://www.fonds-daku.de/

**I have the feeling that this solution might be the easiest to implement, when switching order for controls and slider within the carousel. Maybe with a column-reverse to have the controls on the bottom visually for touch devices.** 


### Second approach:

There is a skip link before slider controls to skip this part
slider controls are the second element to focus. afterwards teasers (typical carousel behavior )
all teasers are accessible if not skipped 
disadvantage: the user can't skip easily if they are e.g. at item 30 of 50
Example: e.g. Medientipps with 25 Slides: https://www.zlb.de/

Let us know what you think :)

**Comment by hirsch88** [comment]
We would suggest that you could implement a skip button like you already did for the website
